express-rest-api/test/index.test.js

```diff
@@ -0,0 +1,11 @@
+const request = require('supertest');
+const app = require('../server/app.js');
+
+describe('GET /', function() {
+  it('return index page', function() {
```

Comment:
It is not a page. Please provide better context for an API server.

Reply:
how about `it('return json response', function() {` ? Is this more clear? 😃